libs/SalesforceSDKCore/SalesforceSDKCoreTests/SFSecurityTests.m

```diff
@@ -79,90 +79,65 @@ - (void)testKeyStoreThreadSafety
     }
 }
 
-#pragma mark - Passcode change tests
-- (void)testNoPasscodeToPasscode
-{
-    // set up the generated keystore
-    SFEncryptionKey *origEncKey = [mgr keyWithRandomValue];
-    [mgr storeKey:origEncKey withKeyType:SFKeyStoreKeyTypeGenerated label:@"origKey"];
-    XCTAssertFalse([mgr.passcodeKeyStore keyStoreAvailable], @"Passcode key store should not be ready.");
-    
-    // add a passcode key to the dictionary so that it is migrated over
-    SFEncryptionKey *passcodeEncKey = [mgr keyWithRandomValue];
-    [mgr storeKey:passcodeEncKey withKeyType:SFKeyStoreKeyTypePasscode label:@"aPasscodeKey"];
-    
-    // now set a passcode
-    NSString *newPasscode = @"IAddedAPasscode!";
-    [[SFPasscodeManager sharedManager] changePasscode:newPasscode];
-    XCTAssertTrue([mgr.passcodeKeyStore keyStoreAvailable], @"Passcode key store is not ready.");
-    
-    // data should have been migrated to passcode keystore
-    [self assertKeyForDictionary:mgr.passcodeKeyStore.keyStoreDictionary
-                       withLabel:[mgr.passcodeKeyStore keyLabelForString:@"aPasscodeKey"]
-                      hasKeyType:SFKeyStoreKeyTypePasscode
-                hasEncryptionKey:passcodeEncKey];
-    
-    // make sure generated-specific key is still in the generated keystore
-    [self assertKeyForDictionary:mgr.generatedKeyStore.keyStoreDictionary
-                       withLabel:[mgr.generatedKeyStore keyLabelForString:@"origKey"]
-                      hasKeyType:SFKeyStoreKeyTypeGenerated
-                hasEncryptionKey:origEncKey];
-}
+#pragma mark - Upgrade tests
 
-- (void)testPasscodeToNoPasscode
+- (void)testUpgradeTo60
```

Comment:
The upgrade test

Reply:
NB: I also tested manually (using SmartSyncExplorerReactNative on 5.3, then changing package.json to point to my fork/branch).